pkg/egressgateway/manager.go

```diff
@@ -39,21 +45,40 @@ type Manager struct {
 
 	// epDataStore stores endpointId to endpoint metadata mapping
 	epDataStore map[endpointID]*endpointMetadata
+
+	identityAllocator identityCache.IdentityAllocator
```

Comment:
Could you add a comment documenting the field please?

Reply:
Done.